experimental/Schemes/src/Auxiliary.jl

```diff
@@ -88,6 +88,53 @@ function pullback(f::AbsCoveredSchemeMorphism, II::AbsIdealSheaf)
   return PullbackIdealSheaf(f, II)
 end
 
+@doc raw"""
+    total_transform(f::AbsSimpleBlowupMorphism, II::IdealSheaf)
+
+Computes the total transform of an ideal sheaf along a blowup.
+
+In particular, this applies in the toric setting. However, note that
+currently (October 2023), ideal sheaves are only supported on smooth
+toric varieties.
```

Comment:
This reads as if ideal sheaves and total transforms are only available in the toric setting. 
They should be available in more generality for covered schemes.